### PR TITLE
Included writeup for Teams DLP E5 license enforcement

### PR DESCRIPTION
**Pull request by tanaybaldua**
We need to mention the impact of E5 license enforcement that is planned to go live by end of week - 9/23. The new part is just a short description and the existing follow up link will provide more information.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/dlp-microsoft-teams.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ You can extend the Teams DLP policy to cover SharePoint Online and OneDrive for
 Office 365 and Microsoft 365 E3 include DLP protection for SharePoint Online, OneDrive, and Exchange Online. This also includes files that are shared through Teams because Teams uses SharePoint Online and OneDrive to share files.
 
 Support for DLP protection in Teams Chat requires E5.
+    
+If you are a Microsoft 365 E3/A3/G3 tenant and your organization is currently using DLP protection in Teams, the support for the same will end by March 2023 as Teams DLP is available to organizations with a Microsoft 365 E5/A5/G5 subscription. Post March 2023, for any active DLP policies that includes Teams, the policy will not be deleted/removed but the protection for Teams will be deactivated. If the policy includes protection for other workloads like SharePoint Online, OneDrive for Business, and Exchange Online in addition to Teams, the protection will continue to be active for all other workloads except Teams. You will not be able to create any new Teams DLP policy after March 2023. To continue your use of Teams DLP, we recommend you get a Microsoft 365 E5/A5/G5 subscription. You can contact your Business Administrator for assistance.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-microsoft-teams.md]:
```suggestion
If you are a Microsoft 365 E3/A3/G3 tenant and your organization is currently using DLP protection in Teams, you should note that support for these will end by March 2023 as Teams DLP is available to organizations with a Microsoft 365 E5/A5/G5 subscription. Post March 2023, for any active DLP policies that include Teams, the policy will not be deleted or removed but the protection for Teams will be deactivated. If the policy includes protection for other workloads like SharePoint Online, OneDrive for Business, and Exchange Online in addition to Teams, the protection will continue to be active for all other workloads except Teams. You will not be able to create any new Teams DLP policy after March 2023. To continue your use of Teams DLP, we recommend you get a Microsoft 365 E5/A5/G5 subscription. You should contact your Business Administrator for assistance.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tanaybaldua, kindly commit the the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
@tanaybaldua - Hi, thanks for making this update.  We never publish guidance about what may or may not be coming in the future as dates and plans slip. Can you please re-write this to account for what the licensing/functionality is right now. Then when it does change, it can be updated.

**Comment by tanaybaldua** [comment]
Hi Chris, we have heard your views on the future plan and thus have decided to not have any update done on this page as Iicensing guidance is already present. I will come back to you in case we have to update anything else. thanks for time and support.

**Pull request closed without merging**